dev-docs/source/IssueTracking.rst

```diff
@@ -0,0 +1,132 @@
+=====================
+Mantid Issue Tracking
+=====================
+
+Mantid uses GitHub issues to track new feature and bugfix
+development. The issue tracker can be found at `GitHub
+<https://github.com/mantidproject/mantid/issues>`_
+
+.. contents:: Contents
+    :local:
+
+Workflow
+^^^^^^^^
+
+Every piece of work starts with an open issue. To find issues that are
+assigned to you see the section
+:ref:`dev-docs-issue-tracking-your-issues`. How an issue is closed is
+then dependent on its requirements:
+
+- Code changes required: generate a **pull request**
+  (:ref:`dev-docs-git-workflow-pull-requests`)
+
+  - Make sure the issue is referenced with ``#issue-number``. It will
+    then be closed when the pull request is merged
+- No code changes required: Close the issue, with a comment explaining
+  why the issue should be closed, for example:
+
+  - This is a duplicate of #1234
```

Comment:
May as well add umbrella issues to this list as they often are not closed by a single PR